app/modules/ads/module.js

```diff
@@ -54,6 +54,7 @@ class Ads {
 		context.push('delayModules', biddersDelay);
 		events.on(events.AD_SLOT_CREATED, (slot) => {
 			bidders.updateSlotTargeting(slot.getSlotName());
+			bidders.refreshBids(slot.config);
```

Comment:
It won't be needed if we decide to register `bidWon` event somewhere in `callBidders`

Reply:
Yep, refactored.